Makefile

```diff
@@ -176,6 +180,9 @@ client: ${PREFIX}/bin/notary
 binaries: ${PREFIX}/bin/notary-server ${PREFIX}/bin/notary ${PREFIX}/bin/notary-signer
 	@echo "+ $@"
 
+escrow: ${PREFIX}/bin/escrow
```

Comment:
No strong feelings. It didn't seem necessary as "escrow" isn't quite as generic as "server". I mostly just got tired of typing `notary-*` on the front of everything :-P